packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx

```diff
@@ -188,17 +188,18 @@ describe('Line chart rendering', () => {
     },
   );
 
-  testWithoutWait(
-    'Should render the Line chart with date x-axis data',
-    LineChart,
-    { data: dateChartPoints },
-    container => {
-      // Assert
-      expect(container).toMatchSnapshot();
-    },
-    undefined,
-    beforeAll,
-  );
+  // @FIXME - this is failing on CI / not failing on local machine
```

Comment:
this is getting bit out of control tbh. non deterministic snapshots generated on local vs CI environment. 

<img width="1297" alt="image" src="https://github.com/microsoft/fluentui/assets/1223799/31eb3f8c-e59d-46a4-b1d2-bb99e4e0296a">

Reply:
@srmukher is looking into this.